lib/composer.js

```diff
@@ -12,28 +12,24 @@ export default class Composer {
     this.configChange = atom.config.onDidChange('latex', () => {
       this.rebuildCompleted = new Set()
     })
+    this.outputLookup = new Map()
```

Comment:
Not super important, but this can be moved outside of the constructor (e.g. above);
```js
outputLookup = new Map()

constructor () { // ...
```

If we move it, we should move the `builderRegistry` property as well 🤘